### DISCUSSION TITLE
Access to the FS?

### DISCUSSION THREAD

**Discussion by codr1** [Hieromon/PageBuilder#43] [Q&A]
Hi!  Great work on this library.  
I am trying to build a small project around your FS Page example.  I would like to store some data in a file in the FS.  Is there any way I can have access to it to read / write some data?   I am reticent to create a new instance of LittleFS in case it conflicts with what you already have...

**Comment by codr1** [comment]
...I think I found it.  FlashFile...

**Comment by Hieromon** [comment]
@codr1 Hello, You are correct, the FSPage example starts a FlashFile filesystem instance in the setup function. Your sketch can share this instance.

However, one thing must be noted. Inside the token handler function (HandlerFuncT handler) linked with PageElement, the file (e.g. entry.html) containing the HTML for the FSPage cannot be accessed. At the time of the PageBuiler calls the token handler, those files remain open. There is no problem to access other files for your sketch.
